Google.Api.Gax.Grpc/Gcp/ChannelRef.cs

```diff
@@ -18,15 +19,29 @@ internal sealed class ChannelRef
     {
         private int _affinityCount;
         private int _activeStreamCount;
-        private int _id;
+        internal int _id;
+        internal Guid _channelId;
+        public  DateTime ChannelStartTime { get; set; }
+        public Action<CallInvoker> ChannelPrimer { get; set; }
+        public bool Grace { get; set; }
```

Comment:
Do these three properties really need to be read-write?

And at the very least I'd want "Grace" to have some documentation - the other properties are reasonably self-documenting, but Grace could mean almost anything.